02_reading_spreadsheets/reading_specific_cells.py

```diff
@@ -8,10 +8,10 @@ def get_cell_info(path):
     sheet = workbook.active
     print(sheet)
     print(f'The title of the Worksheet is: {sheet.title}')
-    print(f'The value of {sheet["A2"].value=}')
-    print(f'The value of {sheet["A3"].value=}')
+    print(f'The value of A2 is {sheet["A2"].value}')
+    print(f'The value of A3 is {sheet["A3"].value}')
```

Comment:
Did you have an error running the original code? This code should work in 3.8 and above:

https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging